src/main/java/de/ait/secondlife/constants/OfferStatus.java

```diff
@@ -1,4 +1,4 @@
-package de.ait.secondlife.domain.constants;
+package de.ait.secondlife.constants;
```

Comment:
It looked fine to me even in the domain folder) but OK - let it be a higher level